tests/plugins/ml/ml/models.yaml

```
@@ -51,6 +51,37 @@ chapters:
         hits:
           hits:
             - _score: 1
+  - synopsis: Search model.
+    path: /_plugins/_ml/models/_search
+    method: POST
+    request:
+      payload:
+        query:
+          match_all: {}
+        size: 1000
+    response:
+      status: 200
+      payload:
+        hits:
+          hits:
+            - _score: 1
+  - synopsis: Get model.
+    path: /_plugins/_ml/models/{model_id}
+    method: GET
+    parameters:
+      model_id: ${get_completed_task.model_id}
+    response:
+      status: 200
+  - synopsis: Update model.
+    path: /_plugins/_ml/models/{model_id}
+    method: PUT
+    parameters:
+      model_id: ${get_completed_task.model_id}
+    request:
+      payload:
+        name: updated_name
```

Nit: this looks too much like a symbol/variable.

```suggestion
        name: updated
```